src/main/java/seedu/address/storage/AddressBookStorage.java

```diff
@@ -5,10 +5,11 @@
 import java.util.Optional;
 
 import seedu.address.commons.exceptions.DataConversionException;
-import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.PersonBook;
+import seedu.address.model.ReadOnlyPersonBook;
 
 /**
- * Represents a storage for {@link seedu.address.model.AddressBook}.
+ * Represents a storage for {@link PersonBook}.
  */
 public interface AddressBookStorage {
 
```

Comment:
May need to change class/method names if AddressBook is deleted to avoid confusion